bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -83,7 +83,7 @@ spec:
           cp -f /opt/cni/bin/* /host/opt/cni/bin/
 
           # Launch the network process
-          exec /usr/bin/openshift-sdn --config=/config/sdn-config.yaml  --url-only-kubeconfig=/etc/kubernetes/kubeconfig --v=${DEBUG_LOGLEVEL:-2}
+          exec /usr/bin/openshift-sdn --config=/config/sdn-config.yaml --proxy-config=/config/kube-proxy-config.yaml --url-only-kubeconfig=/etc/kubernetes/kubeconfig --v=${DEBUG_LOGLEVEL:-2}
```

Comment:
don't you need to remove `--config` too?

Reply:
Sigh, too much back-and-forth. Was thinking I couldn't remove that because openshift-sdn still required it, but of course it doesn't at this point.